distribute.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+set -euo pipefail
+IFS=$'\n\t'
+mkdir dist
+echo "Building Way Cooler..."
```

Comment:
> If you want to test actually uploading these as github releases, I think you are taking the testing a step too far (we're testing our stuff works, not github's ;P). If you want you can extend these scripts to update the github releases, but right now that seems out of the scopes of these two scripts (and would be bettter suited to putting in a third script)

This was the original idea of `ci.py` - provide cross-compiled binary releases automatically. Now it seems that providing GitHub distributions isn't _as_ important if the desired installation is piping a build-from-source script into `sh`.
One thing to note is that this is a "build from source" setup, and the idea was `ci.py` could handle cross-compilation to several different binary targets. Given that we don't have a lot of users, we have a large overlap with the Rust community right now, and we're still in alpha we're probably fine with this distribution, but the goal was to cross-compile and statically link everything via Travis so we could automatically provide binary downloads and distance ourselves from hard Rust/cargo dependencies.